runbotics/runbotics-scheduler/src/websocket/bot/process-launch/bot-process-instance.service.utils.ts

```diff
@@ -6,7 +6,7 @@ const COMMON_PROCESS_INSTANCE_FIELDS = [
 ];
 
 const COMPLETED_PROCESS_INSTANCE_FIELDS = ['status', 'updated', 'output', ...COMMON_PROCESS_INSTANCE_FIELDS];
-const STARTING_PROCESS_INSTANCE_FIELDS = ['input', 'created', ...COMMON_PROCESS_INSTANCE_FIELDS];
+const STARTING_PROCESS_INSTANCE_FIELDS = ['created', ...COMMON_PROCESS_INSTANCE_FIELDS];
```

Comment:
when is `input` set? when `INITIALIZING` message will arrive?

Reply:
Input is set during InProgress, here it was empty